resources/lib/functions.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+""" Functions code """
+
+from __future__ import absolute_import, division, unicode_literals
+
+import logging
+
+from resources.lib import kodilogging, kodiutils
+from resources.lib.modules.iptvsimple import IptvSimple
+
+kodilogging.config()
+_LOGGER = logging.getLogger(__name__)
+
+
+# TODO: throws the error:
+# RuntimeError: No valid addon id could be obtained. None was passed and the script wasn't executed in a normal xbmc manner.
```

Comment:
Note to self: Zou moeten kunnen als de eigen addon id wordt meegegeven met Addon()